### ISSUE TITLE
Use without pushing directly to master/main?

### ISSUE THREAD

**Issue by DevRCRun** [absolute-version/commit-and-tag-version#126]
Similar question to this issue in the original repo.

https://github.com/conventional-changelog/standard-version/issues/817

We also have a policy that we cannot not push directly to master without going through an approval cycle (PR), but are looking at this tool for its simplicity over other options.

Is this use case compatible?

**Comment by TimothyJones** [comment]
Yes, if you run it via automation (with a token that is allowed to push to master).

Tags don’t work well from PRs, generally.

if you want to do pr approved releases, you can have a look at release-please

**Comment by TimothyJones** [comment]
Another option could be to run it with —skip-tags, then raise a PR (with some automation to detect the version bump and tag master after merge).

Either way, I think you’ll need something custom to get around this restriction
